COMMITTING.md

```diff
@@ -0,0 +1,76 @@
+# WHATWG committer guidelines
+
+## General
+
+Each change needs to result in a single commit on the master branch, with no merge commits. The green squash and merge button is OK to use, but be sure to tidy up the commit message per the below guidelines. In particular, if you use the squash functionality, delete the parenthetical pull request number that GitHub adds.
+
+## Commit messages
+
+_(Some of this is derived from [the erlang/otp wiki](https://github.com/erlang/otp/wiki/Writing-good-commit-messages).)_
+
+### Purpose
+
+Good commit messages serve at least three important purposes:
+
+* To help pull requests reviewers understand what is being changed.
+* To make it easy and pleasant to review the history of the repository.
+* To help future maintainers find out why a particular change was made.
+
+As such, you should write a commit message that will allow your future self to understand the intent of change without looking at the diff. This will not only benefit the maintainers, but also the reviewers. You should express the intent of the change, without focusing on the implementation details.
+
+For more on this topic, see ["On commit messages"](http://who-t.blogspot.com/2009/12/on-commit-messages.html).
+
+### Structure and conventions
+
+Commit messages consist of a single title line, a blank line, and then a more detailed change description. The description (and preceding blank line) may be omitted for simple fixes.
+
+All lines should be at most 72 characters, to make them easier to view in GitHub and other tools. (Some sources recommend restricting the title line to 50 characters, but we do not enforce this.)
+
+The title must be written in imperative mode, as if you were commanding someone. This means using verb conjugations like "fix"/"add"/"change" instead of "fixed"/"added"/"changed" or "fixing"/"adding"/"changing". We are less consistent with the description, but often use present tense ("This change adds...").
```

Comment:
Remove subjective:

`The title must be written in imperative mode, as if commanding someone. This means using verb conjugations like _"fix"_/_"add"_/_"change"_ instead of _"fixed"_/_"added"_/_"changed"_ or _"fixing"_/_"adding"_/_"changing"_. We are less consistent with the description, but often use present tense _(i.e. "This change adds…")_.